### PR TITLE
feat: add documentation in instructor dashboard

### PR DESCRIPTION
**Pull request by BryanttV**
### Description
This PR adds the documentation to the instructor dashboard.

### How to Test
1. Install this branch in your environment
2. Go to the instructor dashboard
3. Select the forum notifications tab
4. You should see the added documentation

### Screenshot
image

### CODE REVIEW

File: platform_plugin_forum_email_notifier/static/js/forum_notifier.js [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
+$(function () {
+  const cssUrl = "https://cdn.jsdelivr.net/npm/@edx/paragon@20.45.5/dist/paragon.min.css";
+
+  $("<link>")
+    .attr({
+      href: cssUrl,
+      rel: "stylesheet",
+    })
+    .appendTo("head");
+});
```
Comment (Ian2012) [inline platform_plugin_forum_email_notifier/static/js/forum_notifier.js:45]: why are you adding this CC at the head?

why not just adding the css ref?

File: platform_plugin_forum_email_notifier/static/html/forum_notifier.html [hunk 2]
```diff
@@ -1,15 +1,35 @@
...
+                {% trans "All posts (Daily digest): I want to receive email notifications for all posts in a daily digest." %}
+            </li>
+            <li>
+                {% trans "All posts (Weekly digest): I want to receive email notifications for all posts in a weekly digest." %}
+            </li>
+        </ul>
+    </div>
 
-    <select class="email-notifications-select">
+    <p><b>{% trans "These notifications are only sent to the instructor of the course." %}</b></p>
```
Comment (Ian2012) [inline platform_plugin_forum_email_notifier/static/html/forum_notifier.html:21]: Can you remove this part?

The notification are also sent to students but students doesn't have email preferences as of now

**Review by Ian2012 — COMMENTED** [review]
LGTM, just some nits

**Review by mariajgrimaldi — APPROVED** [review]
This looks good! Thanks. Keep in mind we'll probably need the PO approval later on (he's on PTO) but I don't think we'll need to change anything.

**Review by Ian2012 — APPROVED** [review]

**Pull request merged**